### ISSUE TITLE
Review JMX limitations on Community and how to get the same data available

### ISSUE THREAD

**Issue by lcabaceira** [lcabaceira/crystal#4]
We need to check what are the JMX limitations on the Community edition. The idea was to make something 100% community friendly. Features that are relying on Enterprise should be planned for the future.

**Comment by ohej** [comment]
There is nothing wrong with having enterprise only features in there, but the tests should degrade gracefully if it's on community. 

One example is if there are Enterprise specific metrics available in Enterprise, but you're running on community these metrics are just not available.

**Comment by lcabaceira** [comment]
Agree, but we should focus more on the community aspects as this early stage and then we can add those features at a later stage, as the sources are available and exposed.
